examples/desktop/package.json

```diff
@@ -15,6 +15,24 @@
       "width": 900,
       "height": 500,
       "minWidth": 500
+    },
+    "build": {
```

Comment:
1. can `examples/desktop` keep the appling config + icons? I'm using this pear app for testing $ pear build, when you seed it's important they are in the drive.

2. `pear init ui`: where's this repo at? also needs to come with package.json::`pear.build` + `assets/` (appling icons)